drill.cpp

```diff
@@ -556,12 +578,26 @@ void ExcellonProcessor::export_ngc(const string of_dir, const boost::optional<st
     cout << "Exporting milldrill... " << flush;
 
     zchange << setprecision(3) << fixed << target->zchange * cfactor;
-    tiling->setGCodeEnd((zchange_absolute ? "G53 " : "") + string("G00 Z") + zchange.str() +
-                         " ( All done -- retract )\n" + postamble_ext +
-                         "\nM5      (Spindle off.)\nG04 P" +
-                         to_string(target->spindown_time) +
-                        "\nM9      (Coolant off.)\n"
-                         "M2      (Program end.)\n\n");
+    
+    stringstream gcode_end;
+    gcode_end << setprecision(3) << fixed;
+    
+    if (zchange_absolute) {
+        gcode_end << "G53 ";      
+    }
+    
+    gcode_end << "G0 Z" << target->zchange * cfactor;
```

Comment:
Did we already compute `zchange` above?  So we shouldn't need `target->zchange * cfactor` again, right?

Reply:
You're correct, this can be eliminated now.